### ISSUE TITLE
MAE Training

### ISSUE THREAD

**Issue by ehsanngh** [laumerf/4DHeartModel#7]
Hello, and thank you for making the code publicly available.

I'm trying to train the models and ran into an issue when training the MAE. After ~160 epochs, with the validation loss of 9.83987683e-06 and the learning rate reduced to 5.63846e-06, the phase plot still shows no correlation, even though the frequency and EF plots do show some correlation (see below).

Do you have any insight into why this might be happening or suggestions on how to move forward?

Thanks for your time.

<img width="579" height="471" alt="Image" src="https://github.com/user-attachments/assets/f3c965bb-8e0c-4fd4-8ba1-77fa3b561ee7" />
<img width="583" height="471" alt="Image" src="https://github.com/user-attachments/assets/31937488-da78-4921-a388-807ec5509eda" />
<img width="592" height="471" alt="Image" src="https://github.com/user-attachments/assets/20e03ac9-1ef5-4d75-9c6d-a4c3d96a79d7" />

**Comment by whucsu** [comment]
> Hello, and thank you for making the code publicly available.
> 
> I'm trying to train the models and ran into an issue when training the MAE. After ~160 epochs, with the validation loss of 9.83987683e-06 and the learning rate reduced to 5.63846e-06, the phase plot still shows no correlation, even though the frequency and EF plots do show some correlation (see below).
> 
> Do you have any insight into why this might be happening or suggestions on how to move forward?
> 
> Thanks for your time.
> 
> <img alt="Image" width="579" height="471" src="https://private-user-images.githubusercontent.com/59973930/524966175-f3c965bb-8e0c-4fd4-8ba1-77fa3b561ee7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzUxMTg5MDIsIm5iZiI6MTc3NTExODYwMiwicGF0aCI6Ii81OTk3MzkzMC81MjQ5NjYxNzUtZjNjOTY1YmItOGUwYy00ZmQ0LThiYTEtNzdmYTNiNTYxZWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDAyVDA4MzAwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwN2M3ZjUyYjVhNGQ0ZGQ2ZjgzYTA4M2MzMTVkMDM1ODI4YzA1MDAxNTdhZGM5ZGJkOGE5YTVkM2E4Y2EwOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.fcEYDS3AZ9Ctj9AJReFJOXaodNjDjhmman_NqFw70BE"> <img alt="Image" width="583" height="471" src="https://private-user-images.githubusercontent.com/59973930/524966174-31937488-da78-4921-a388-807ec5509eda.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzUxMTg5MDIsIm5iZiI6MTc3NTExODYwMiwicGF0aCI6Ii81OTk3MzkzMC81MjQ5NjYxNzQtMzE5Mzc0ODgtZGE3OC00OTIxLWEzODgtODA3ZWM1NTA5ZWRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDAyVDA4MzAwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2MjhjODQ3NWUzMDIwZjU1NjQ1MTAzYzNlYjYyMTA2MWFkMTM5MzZiOThhOGY3NmE2Yjk3ZWM0MmMyOWQxMjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Uj-oNpLZAytMMwngyUyNLtqc7RHK5usOslcj9vbhnjg"> <img alt="Image" width="592" height="471" src="https://private-user-images.githubusercontent.com/59973930/524966176-20e03ac9-1ef5-4d75-9c6d-a4c3d96a79d7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzUxMTg5MDIsIm5iZiI6MTc3NTExODYwMiwicGF0aCI6Ii81OTk3MzkzMC81MjQ5NjYxNzYtMjBlMDNhYzktMWVmNS00ZDc1LTljNmQtYTRjM2Q5NmE3OWQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDAyVDA4MzAwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwMTlhZGE2NjkwNzM2MzQzM2ZiMjg1YTNhYzI0YzJjM2JmZjY5MTQ5NzQ5MTI4ODVlMGI4NDdiM2FkOTBhODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rz6dCEpR8K50AkANAO8GyoJ1bAN_AX7orFkBj153p_M">

I still can not reproduce the whole process due to different  enviroment and gpu.
would like to share your requirements and the modification of any configure files like yaml ?
thanks
